install/automationservicebroker/deploy-automation-broker-apb.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: automation-broker-apb
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: automation-broker-apb
+    namespace: "${NAMESPACE}"
+
+- apiVersion: rbac.authorization.k8s.io/v1beta1
+  kind: ClusterRoleBinding
+  metadata:
+    name: automation-broker-apb
+  roleRef:
+    name: cluster-admin
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+  subjects:
+  - kind: ServiceAccount
+    name: automation-broker-apb
+    namespace: "${NAMESPACE}"
+
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: automation-broker-apb
+  spec:
+    backoffLimit: 5
+    activeDeadlineSeconds: 100
+    template:
+      spec:
+        restartPolicy: OnFailure
+        serviceAccount: automation-broker-apb
+        containers:
+        - image: docker.io/automationbroker/automation-broker-apb:latest
+          name: automation-broker-apb
+          command: [ "entrypoint.sh", "provision", "-e", "broker_name=automation-service-broker", "-e", "create_broker_namespace=true" ]
```

Comment:
you could use `args:` instead of command so you don't have to specify `entrypoint.sh`.

Reply:
Is there a reason why we are going with `automation-service-broker` instead of `automation-broker` (default)?

Reply:
If you are taking the `$NAMESPACE` as an argument you may want to add `-e broker_namespace=${NAMESPACE}`

Reply:
ACK on first.
Second, that's what I was told to set it to to be similar to template-service-broker.
Third, as mentioned above the NAMESPACE is where the apb is running not the broker.